fluid-project/hearth
# feat: add the ability for users to add resources to their collections

Adds Livewire support to Hearth, which is used to build a Livewire component for selecting resources users want to add to their resource collections. Also, updates ResourceCollection controller and requests to have ability to include resources on resource collection create, and update. 

**Testing**

- Verify that ResourceSelect component loads all available resources
- Verify that ResourceSelect component loads any resources those are already included in a collection
- Verify adding or removing resources from ResourceSelect
